packages/app/components/swipe-list.web.tsx

```diff
@@ -89,25 +89,23 @@ export const SwipeList = ({
         e.activeIndex + 1 < data.length ? e.activeIndex + 1 : undefined,
       ];
       if (isSwipeListScreen) {
-        scrollTimer.current = setTimeout(() => {
-          router.replace(
-            {
-              pathname: "/profile/[username]/[dropSlug]",
-              query: {
-                ...router.query,
-                initialScrollIndex: activeIndex,
-                username: data[activeIndex].creator_username,
-                dropSlug: data[activeIndex].slug,
-              },
+        router.replace(
+          {
+            pathname: "/profile/[username]/[dropSlug]",
+            query: {
+              ...router.query,
+              initialScrollItemId: data[e.activeIndex].nft_id,
```

Comment:
root issue was combination of setTimeout and using activeIndex state instead of e.activeIndex here. Other changes in the PR are for replacing index with nft_id

Reply:
The timer most likely had a purpose, right? Are we running into a regression maybe?

Reply:
Yeah, the timer is for fixing the CSS swipe list flicker issue. But this is the swipe.js version, so removing or adding it are both good options.